docs/getting_started.Rmd

```diff
@@ -134,108 +131,8 @@ board.pin_meta("mtcars")
 
 While we’ll do our best to keep the automatically generated metadata consistent over time, I’d recommend manually capturing anything you really care about in metadata.
 
-
-## 🚧 Versioning
```

Comment:
Instead of removing this section altogether, can we keep the demo for how to write different versions of content to the same pin and how to use the `version` argument?

Reply:
Ah, nice catch! restored!